salt: Remove the Control Plane Ingress controller PDB in pre-upgrade

TeddyAndrieux opened the pull request:
The control plane Ingress might be not working during the upgrade
but since it's only for a specific upgrade path and transient since it's
fixed after the upgrade we only remove the PodDisruptionBugdget so that
the upgrade complete

---

NOTE: The workaround has been removed in 123.0 integration branch

TeddyAndrieux committed:
salt: Remove the Control Plane Ingress controller PDB in pre-upgrade

NOTE: The control plane Ingress may be not working during the upgrade
but since it's only for a specific upgrade path and transient since it's
fixed after the upgrade we only remove the PodDisruptionBugdget so that
the upgrade complete

bert-e commented:
# Hello teddyandrieux,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/metalk8s/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/fix-cp-ingress-upgrade` into
`development/2.11`
* [w/123.0/bugfix/fix-cp-ingress-upgrade](https://github.com/scality/metalk8s/compare/development/123.0...w/123.0/bugfix/fix-cp-ingress-upgrade) will be merged into `development/123.0` 



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @chengyanjin

* @lucieleonard

* @jbwatenbergscality

* @alexandre-allard-scality

* @gdemonet

* @teddyandrieux

* @nicolast

* @thomasdanan

TeddyAndrieux committed:
packages: Disable repo_gpgcheck for Kubernetes repository

Right now we cannot use the Kubernetes with repo_gpgcheck enabled likely
due to a bug.
Since we only use this repository at build time and we still enable the
gpgcheck for package it self we can safely disable repo_gpgcheck, it
will avoid this kind of issue in the future (NOTE: We already faced the
exact same issue one year ago)

bert-e commented:
# History mismatch



Merge commit #dd90e7ecd498e3f4a9b31c8e8d98a7c7dc032520 on the integration branch
`w/123.0/bugfix/fix-cp-ingress-upgrade` is merging a branch which is neither the current
branch `bugfix/fix-cp-ingress-upgrade` nor the development branch
`development/123.0`.

It is likely due to a rebase of the branch `bugfix/fix-cp-ingress-upgrade` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

bert-e commented:
# Lossy reset warning



There seems to be *manual* commits on integration branches (e.g. conflict
resolutions) that will be lost if you chose to *reset*.

**You can use the `force_reset` command if you still want me
to delete those branches.**

bert-e commented:
# History mismatch



Merge commit #dd90e7ecd498e3f4a9b31c8e8d98a7c7dc032520 on the integration branch
`w/123.0/bugfix/fix-cp-ingress-upgrade` is merging a branch which is neither the current
branch `bugfix/fix-cp-ingress-upgrade` nor the development branch
`development/123.0`.

It is likely due to a rebase of the branch `bugfix/fix-cp-ingress-upgrade` and the
merge is not possible until all related `w/*` branches are deleted or updated.

**Please use the `reset` command to have me reinitialize these branches.**

TeddyAndrieux commented:
/force_reset

bert-e commented:
# Reset complete



I have successfully deleted this pull request's integration branches.

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/fix-cp-ingress-upgrade` into
`development/2.11`
* [w/123.0/bugfix/fix-cp-ingress-upgrade](https://github.com/scality/metalk8s/compare/development/123.0...w/123.0/bugfix/fix-cp-ingress-upgrade) will be merged into `development/123.0` 



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`




You can set option `create_pull_requests` if you need me to create
**integration pull requests** in addition to integration branches, with:

```
@bert-e create_pull_requests
```

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* one peer





Peer approvals *must* include at least 1 approval from the following list:

* @chengyanjin

* @lucieleonard

* @jbwatenbergscality

* @alexandre-allard-scality

* @gdemonet

* @teddyandrieux

* @nicolast

* @thomasdanan

TeddyAndrieux commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.11`

* :heavy_check_mark: `development/123.0`



The following branches will **NOT** be impacted:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.11`

* :heavy_check_mark: `development/123.0`



The following branches have **NOT** changed:

* `development/2.0`
* `development/2.1`
* `development/2.10`
* `development/2.2`
* `development/2.3`
* `development/2.4`
* `development/2.5`
* `development/2.6`
* `development/2.7`
* `development/2.8`
* `development/2.9`



Please check the status of the associated issue None.

Goodbye teddyandrieux.
